daphne_worker/src/roles/aggregator.rs

```diff
@@ -465,31 +464,28 @@ impl<'srv> DapAggregator<DaphneWorkerAuth> for DaphneWorker<'srv> {
         .flatten()
         .collect::<HashSet<ReportId>>();
 
-        try_join_all(agg_store_request_data.into_iter().map(
-            |(agg_store_name, out_shares)| async {
-                // Only aggregate the output shares that haven't been replayed.
-                let agg_share = DapAggregateShare::try_from_out_shares(
-                    out_shares
-                        .into_iter()
-                        .filter(|out_share| !replayed.contains(&out_share.report_id)),
-                )?;
-
-                std::result::Result::<_, DapError>::Ok(
+        // Only aggregate the output shares if none are replayed
+        if replayed.is_empty() {
+            // TODO: don't produce a vec
```

Comment:
When you add a TODO like this, it's useful to add your GitHub handle so that others can follow up with you about what you meant.